### DISCUSSION TITLE
右侧的更新日历有问题。

### DISCUSSION THREAD

**Discussion by shaoyangshujian** [Tokinx/Wing#34] [讨论]
![屏幕截图 2022-08-26 060219](https://user-images.githubusercontent.com/111892369/186776837-8104fc70-ec3e-4c91-9b4f-29d8143b8e8c.png)
DAYS永远显示1
亮点只有在登陆后台后才显示，退出登录或者是游客访问不显示更新日历的亮点。
但是第一排的统计，无论是否登陆都不会更新。
![屏幕截图 2022-08-26 060425](https://user-images.githubusercontent.com/111892369/186777074-c5c7cadf-e6e7-4124-a829-a28b5ef2f0fd.png)

环境：php7.3（高版本php总是出现莫名其妙问题）

**Comment by Tokinx** [comment]
有网址吗？我看一下

**Reply by shaoyangshujian** [reply]
chngp.com
